### ISSUE TITLE
Error when running munkireport-runner 

### ISSUE THREAD

**Issue by jby** [munkireport/homebrew#7]
This seems to come with the updated (1.9) module at least on Sequoia:
```
    homebrew.py Error: Traceback (most recent call last):
  File "/usr/local/munkireport/scripts/homebrew.py", line 181, in <module>
    main()
  File "/usr/local/munkireport/scripts/homebrew.py", line 165, in main
    result = get_brew_info(brew)
  File "/usr/local/munkireport/scripts/homebrew.py", line 45, in get_brew_info
    bottle["requirements"] = ', '.join(brew["requirements"])
TypeError: sequence item 0: expected str instance, dict found
WARNING: homebrew.py return code: 1
```

**Comment by tuxudo** [comment]
This should now be resolved with version v1.10 of the module.

**Comment by jby** [comment]
Confirmed!
